pkg/serviceaccount/legacy.go

```diff
@@ -112,7 +119,7 @@ func (v *legacyValidator) Validate(tokenData string, public *jwt.Claims, private
 		}
 
 		// Make sure service account still exists (name and UID)
-		serviceAccount, err := v.getter.GetServiceAccount(namespace, serviceAccountName)
+		serviceAccount, err := v.getter.GetServiceAccountWithMultiTenancy(tenant, namespace, serviceAccountName)
 		if err != nil {
 			klog.V(4).Infof("Could not retrieve service account %s/%s: %v", namespace, serviceAccountName, err)
```

Comment:
Minor: Change to ("Could not retrieve service account %s/%s/%s: %v", tenant, namespace, serviceAccountName, err)